engine/src/main/java/org/terasology/logic/players/DebugControlSystem.java

```diff
@@ -140,7 +139,7 @@ public void onKeyDown(KeyDownEvent event, EntityRef entity) {
                 mouseGrabbed = !mouseGrabbed;
                 DebugProperties debugProperties = (DebugProperties) nuiManager.getHUD().getHUDElement("engine:DebugProperties");
                 debugProperties.setVisible(!mouseGrabbed);
-                Mouse.setGrabbed(mouseGrabbed);
+               // Mouse.setGrabbed(mouseGrabbed);
```

Comment:
More mystery commented out code, made an earlier comment on this specifically, still curious :-)